documentation/src/main/asciidoc/reference/introduction.adoc

```diff
@@ -0,0 +1,517 @@
+[[getting-started]]
+= Introduction to Hibernate Reactive
+
+:example: https://github.com/hibernate/hibernate-reactive/tree/master/example
+
+Before you start, we recommend taking a quick look at the example
+program in the {example}[`example`] directory, which shows off all
+the "bits" you'll need to get your own program up and running.
+
+== Setting up a reactive Hibernate project
+
+Setting up a project that uses Hibernate Reactive isn't hard at all.
+
+If you're using Hibernate Reactive outside of the Quarkus environment,
+you'll need to:
+
+- include Hibernate Reactive itself along with the appropriate Vert.x
+  reactive database client as dependencies of your project, and
+- configure Hibernate Reactive with information about your database,
+  using Hibernate configuration properties.
+
+Or, if you're using Hibernate Reactive in Quarkus.... TODO
```

Comment:
Folks, I could use some suggestions here!

Reply:
For quarkus one needs to use the proper extension. You can check this quickstart: https://github.com/quarkusio/quarkus-quickstarts/blob/master/hibernate-reactive-quickstart/pom.xml#L35

Maybe we can link to the quikcstart here.

Reply:
Well the information there doesn't really help someone who doesn't already know a lot about Quarkus know what they have to do to use HR in Quarkus. I want a brief outline of the basic steps.

Reply:
Yes, but this is an introduction, I would just add the extension needed and mention that they exist plus the properties needed to set in the application.properties. I don't think there is much more to do in term of configuration.
I think I would leave a complete guide to configure a project with Quarkus to a separate section.

Reply:
Well, that's almost the exact opposite of the information I would like to have in this preamble. I don't want specific details of this kind, I want a broad-brush picture of the *process* I need to follow.

Reply:
I mean, for quarkus the steps are the same: you need to add the dependencies (Hibernate reactive and driver) and configure the application. The only difference is that you use different coordinates. I'm not sure how this is helpful without the exacts coordinates but I'm fine with it.